modules/statics/src/coins.ts

```diff
@@ -100,14 +100,7 @@ export const coins = CoinMap.fromCoins([
   account('tdot', 'Testnet Polkadot', Networks.test.dot, 12, UnderlyingAsset.DOT, DOT_FEATURES, KeyCurve.Ed25519),
   account('eth', 'Ethereum', Networks.main.ethereum, 18, UnderlyingAsset.ETH, ETH_FEATURES_WITH_STAKING),
   account('teth', 'Kovan Testnet Ethereum (Deprecated)', Networks.test.kovan, 18, UnderlyingAsset.ETH, ETH_FEATURES),
-  account(
```

Comment:
do we need to delete the entire entry? or can we just use underlyingasset.eth?

Reply:
@wesgraham did not deleted it, only change is asset, since prettier was automatically applied, the details of the GTETH is now in one line instead of multiple lines